Kickstarter-iOS/Features/PledgeOverTime/Views/PledgePaymentPlanOptionView.swift

```diff
@@ -141,20 +217,70 @@ final class PledgePaymentPlanOptionView: UIView {
   @objc private func onOptionTapped() {
     self.viewModel.inputs.optionTapped()
   }
+
+  @objc private func onTermsOfUseButtonTapped() {
+    self.viewModel.inputs.termsOfUseTapped()
+  }
+
+  // MARK: - Functions
+
+  private func setupIncrementsStackView(with increments: [PledgePaymentIncrementFormatted]) {
+    var dateLabels: [UILabel] = []
+    increments.forEach { increment in
+      let incrementStackView = UIStackView()
```

Comment:
That makes sense, and it would definitely help clean things up. I propose handling this as part of a tech-debt ticket so we can prioritize unblocking the API tasks for now. Let me know if that works for you!